### PR TITLE
chore: standardize changelogs

### PR DESCRIPTION
**Pull request by stefano-garzarella**
### Summary of the PR

Resolve small differences between our changelogs and fix small problems.

### Requirements

Before submitting your PR, please make sure you addressed the following
requirements:

- [x] All commits in this PR have Signed-Off-By trailers (with
  `git commit -s`), and the commit message has max 60 characters for the
  summary and max 75 characters for each description line.
- [x] All added/changed functionality has a corresponding unit/integration
  test.
- [x] All added/changed public-facing functionality has entries in the "Upcoming 
  Release" section of CHANGELOG.md (if no such section exists, please create one).
- [x] Any newly added `unsafe` code is properly documented.

### CODE REVIEW

**Review by sandrobonazzola — APPROVED** [review]
LGTM

**Review by sandrobonazzola — APPROVED** [review]

File: vhost-device-scmi/CHANGELOG.md [hunk 1]
```diff
@@ -9,16 +9,16 @@
 
 ### Deprecated
 
-## [0.2.0]
+## v0.2.0
```
Comment (epilys) [inline vhost-device-scmi/CHANGELOG.md:12]: What do you think about adopting a specific CHANGELOG format like https://common-changelog.org/ ? There seem to be some tools that verify the markdown doc is compliant to specific CHANGELOG formats, but I haven't checked them out myself yet.

Comment (stefano-garzarella) [reply vhost-device-scmi/CHANGELOG.md:12]: I like the idea, in this case I tried to follow https://github.com/rust-vmm/community/blob/main/docs/crate_release.md , so should we update also that ?

I think it is not strict, but having a common thing and maybe integrating the checks into the CI would not be bad.

Comment (epilys) [reply vhost-device-scmi/CHANGELOG.md:12]: Yeah makes sense! I made an issue here: https://github.com/rust-vmm/community/issues/182

**Review by epilys — APPROVED** [review]

**Review by vireshk — APPROVED** [review]

**Pull request merged**